src/components/bulk/BulkRefusjon.tsx

```diff
@@ -22,11 +22,9 @@ export const BulkRefusjon = (props: AnsattID) => {
         feilmelding={a?.beloepError}
         beloep={a?.beloep}
         handleChange={handleChange}
-        label={
-          <div style={{ display: 'flex' }}>
-            Beløp
-            <EksempelBulk />
-          </div>} />)
+        label={<div style={{ display: 'flex' }}>Beløp<HjelpetekstRefusjon /></div>}
```

Comment:
Jeg er ikke noe stor fan av inline styling, selv om det er bare en linje. Jeg mener det blir vanskeligere å feilsøke i og med at koden blir spredt på flere steder.

Reply:
Dette er den foreslåtte løsning fra design komponentene til NAV lik jeg forstår det. Finnes det andre måter å løse dette på?

Se eksemplet i https://design.nav.no/components/hjelpetekst

Reply:
Man må i så fall lage en egen klasse for stylingen

`label={<div className="hjelpetekst-label">Beløp<HjelpetekstRefusjon /></div>}`

Kan hende det bare er jeg som er sær :-)